baseapp/baseapp_pages/graphql/object_types.py

```diff
@@ -95,13 +101,22 @@ def get_node(cls, info, id):
 
     @classmethod
     def get_queryset(cls, queryset, info):
+        MAX_COMPLEXITY = 3
```

Comment:
lets move this `MAX_COMPLEXITY` to be `cls.MAX_COMPLEXITY` instead, this will make this number be easily changed per project